search/src/main/scala/uk/ac/wellcome/platform/api/search/rest/WorksController.scala

```diff
@@ -36,7 +36,7 @@ class WorksController(
           worksService
             .listOrSearch(index, searchOptions)
             .map {
-              case Left(err) => elasticError("Image", err)
```

Comment:
Why was this "Image" previously?

Reply:
Because I made a mistake, and on the search code path it doesn't actually matter.

That `documentType` variable is used to fill in the 404 message `"$documentType not found for ID $id"`, which we don't return for search responses.